src/anthropic/resources/beta/messages/messages.py

```diff
@@ -1315,6 +1322,15 @@ def tool_runner(
             **(extra_headers or {}),
         }
 
+        client_side_tools: list[BetaRunnableTool] = []
+        server_side_tools: list[BetaToolUnionParam] = []
```

Comment:
nit: this is a bit of a misnomer
```suggestion
        runnable_tools: list[BetaRunnableTool] = []
        raw_tools: list[BetaToolUnionParam] = []
```
(not a big fan of `raw_tools` so fft come up with something else)